Source/WebCore/page/DragController.cpp

```diff
@@ -240,15 +241,11 @@ bool DragController::performDragOperation(DragData&& dragData)
     SetForScope isPerformingDrop(m_isPerformingDrop, true);
     RefPtr focusedOrMainFrame = m_page->checkedFocusController()->focusedOrMainFrame();
     if (!focusedOrMainFrame)
-        return false;
+        focusedOrMainFrame = &frame;
```

Comment:
This is a little strange.  Under what conditions would focusedOrMainFrame be null but the caller-passed-in frame (which it seems like is always the local main frame) would not be null?  If this is needed then the variable should be renamed, but I think it should be removed.

Reply:
The case where you have not focussed the subframe you are dragging into.